gum/backend-arm/gumstalker-arm.c

```diff
@@ -305,6 +308,9 @@ static gboolean gum_stalker_iterator_arm_next (GumStalkerIterator * self,
     const cs_insn ** insn);
 static gboolean gum_stalker_iterator_thumb_next (GumStalkerIterator * self,
     const cs_insn ** insn);
+static gboolean gum_is_exclusive_store_insn (const cs_insn * insn);
+static void gum_increment_exclusive_load_offset (GumGeneratorContext * gc);
+static gboolean gum_is_exclusive_load_insn (const cs_insn * insn);
```

Comment:
The `load` and `store` helpers should go to the very bottom, after `gum_stalker_is_kuser_helper`. I'd also order them `load` and then `store`.